packages/openactive-integration-tests/test/features/notifications/change-of-logistics-notifications/implemented/change-of-logistics-notifications-test.js

```diff
@@ -0,0 +1,90 @@
+  C1FlowStage,
+  C2FlowStage,
+  FlowStageUtils,
+  TestInterfaceActionFlowStage,
+  OrderFeedUpdateFlowStageUtils,
+  BFlowStage,
+} = require('../../../../helpers/flow-stages');
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'notifications',
+  testFeature: 'change-of-logistics-notifications',
+  testFeatureImplemented: true,
+  testIdentifier: 'change-of-logistics-notifications',
+  testName: 'Updating logistics information in after B request.',
+  testDescription: 'ChangeOfLogisticsSimulateAction triggered after B request to update name, startDate, endDate, duration or location properties of Opportunity.',
+  // The primary opportunity criteria to use for the primary OrderItem under test
+  testOpportunityCriteria: 'TestOpportunityBookable',
+  controlOpportunityCriteria: 'TestOpportunityBookable',
+},
+(configuration, orderItemCriteriaList, featureIsImplemented, logger) => {
+  const requestHelper = new RequestHelper(logger);
+
+  // ## Initiate Flow Stages
+  const defaultFlowStageParams = FlowStageUtils.createDefaultFlowStageParams({ requestHelper, logger });
+  const fetchOpportunities = new FetchOpportunitiesFlowStage({
+    ...defaultFlowStageParams,
+    orderItemCriteriaList,
+  });
+  const c1 = new C1FlowStage({
+    ...defaultFlowStageParams,
+    prerequisite: fetchOpportunities,
+    getInput: () => ({
+      orderItems: fetchOpportunities.getOutput().orderItems,
+    }),
+  });
+  const c2 = new C2FlowStage({
+    ...defaultFlowStageParams,
+    prerequisite: c1,
+    getInput: () => ({
+      orderItems: fetchOpportunities.getOutput().orderItems,
+    }),
+  });
+  const b = new BFlowStage({
+    ...defaultFlowStageParams,
+    prerequisite: c2,
+    getInput: () => ({
+      orderItems: fetchOpportunities.getOutput().orderItems,
+      totalPaymentDue: c2.getOutput().totalPaymentDue,
+    }),
+  });
+  const [simulateChangeOfLogisticsUpdate, orderFeedUpdate] = OrderFeedUpdateFlowStageUtils.wrap({
+    wrappedStageFn: prerequisite => (new TestInterfaceActionFlowStage({
+      ...defaultFlowStageParams,
+      testName: 'Simulate Change of Logistics Update (Test Interface Action)',
+      prerequisite,
+      createActionFn: () => ({
+        type: 'test:ChangeOfLogisticsSimulateAction',
+        objectType: 'Order',
+        objectId: b.getOutput().orderId,
+      }),
+    })),
+    orderFeedUpdateParams: {
+      ...defaultFlowStageParams,
+      prerequisite: b,
+      testName: 'Orders Feed (after Simulate Change of Logistics Update)',
+    },
+  });
+
+  // ## Set up tests
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(fetchOpportunities);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(c1);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(c2);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(b);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(simulateChangeOfLogisticsUpdate);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(fetchOpportunities, () => {
```

Comment:
@nickevansuk @lukehesluke 
So my plan was to check feeds again as ChangeOfLogisticsSimulateAction updates information in Opportunity feeds, but wasn't sure how to do it.
Is this something that can be done?
I check manually that information is updated, but not sure how to do it here.